zetaclient/chainclient.go

```diff
@@ -10,8 +10,8 @@ import (
 // general chain client
 
 type ChainClient interface {
-	GetLastBlockHeight() int64 // 0 means error
-	SetLastBlockHeight(int64)
+	//GetLastBlockHeight() int64 // 0 means error
+	//SetLastBlockHeight(int64)
```

Comment:
I think we should remove methods in interfaces no longer used instead of commenting them out if we plan to no longer use them. It might be confusing when reading the code.